ocpp/messages.py

```diff
@@ -203,7 +233,8 @@ def to_json(self):
         ],
             # By default json.dumps() adds a white space after every separator.
             # By setting the separator manually that can be avoided.
-            separators=(',', ':')
+            separators=(',', ':'),
+            cls=_DecimalEncoder
```

Comment:
I'm wondering if this should just be a constant at the top to reduce some lines of code. Not really a huge deal though.

Reply:
What do you mean? I am afraid I don't get it.

Reply:
Ah nevermind. I miss read the code. I thought that was a list you were passing in as a param, not params to the function. Disregard.